web/src/components/DandisetList.vue

```diff
@@ -4,12 +4,13 @@
     subheader
   >
     <v-list-item
-      v-for="item in dandisets"
+      v-for="(item, index) in dandisets"
       :key="item.dandiset.identifier"
       selectable
       :to="{
         name: 'dandisetLanding',
-        params: { identifier: item.dandiset.identifier, origin }
+        params: { identifier: item.dandiset.identifier, origin },
+        query: { ...$route.query, pos: getPos(index) },
```

Comment:
I think some more query parameters need to be passed here.

Notice, you can filter the dandiset listing page by empty/draft dandisets.
![Screenshot from 2022-07-22 13-23-50](https://user-images.githubusercontent.com/37340715/180492217-65657db1-7e08-4449-bcef-07eafafa2867.png)

Changing those controls changes the values of the `draft` and `empty` query parameters used in the `/dandisets` endpoint call, so you'll need to pass those values to the DLP as well and use them when determining the next/previous dandiset.

Reply:
Ah, got it!